htpy/html2htpy.py

```diff
@@ -147,6 +147,9 @@ def __init__(
 
     @property
     def python_element_name(self) -> str:
+        # Note: <del> tags should be translated to "del_" instead of "del" in htpy.
+        if self.html_tag == "del":
```

Comment:
```suggestion
        if keyword.iskeyword(self.html_tag):
            return self.html_tag + "_"
```

I guess `del` is the only builtin html element that clashes with python keywords. But keyword.iskeyword() is a more general approach that avoids the problem altogether now+in the future if new keywords are added to python. we use the same approach for attributes in this file so the keyword module is already imported: https://docs.python.org/3/library/keyword.html

Reply:
I agree! That is a lot cleaner! Fixed!